_inc/client/traffic/ads.jsx

```diff
@@ -27,7 +27,12 @@ export const Ads = moduleSettingsForm(
 		 */
 		getInitialState() {
 			return {
-				enable_header_ad: this.props.getOptionValue( 'enable_header_ad', 'wordads' )
+				enable_header_ad: 			this.props.getOptionValue( 'enable_header_ad', 'wordads' ),
```

Comment:
our eslint rules for [key-spacing](https://eslint.org/docs/rules/key-spacing) actually prefers that there's only one space after each colon.  No need to keep these aligned.